ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ErrorCodes.java

```diff
@@ -24,7 +24,6 @@ private ErrorCodes() {
   }
 
   public static final int ERROR_CODE_MISSING_PARAM = 1;
-  public static final int ERROR_CODE_UNKNOWN_PARAM = 2;
```

Comment:
Hey @purplefox , thanks for the review! I discovered more code related to deserializing with unknown fields that I've removed in my latest commit on this PR. Hopefully it still looks good?

It's a bit odd that there's a skipped number in these error codes now. Do you think it's better to leave it in and just mark it as unused?

Reply:
I am actually thinking maybe we should unify these with the old API error codes. It seems a bit odd to have two sets of codes. Perhaps we should add an issue to track this and revisit it before we release the client and new API?

Reply:
Makes sense, here's an issue: https://github.com/confluentinc/ksql/issues/5249
I've added it to the phase 2 epic and marked it for 0.10.0.